example/cmd/device-simple/res/configuration.toml

```diff
@@ -65,3 +65,7 @@ File = "./device-simple.log"
     [DeviceList.Protocols.other]
       Address = "simple01"
       Port = "300"
+  [[DeviceList.AutoEvents]]
```

Comment:
It's not clear to me with the given syntax how someone would add more than one device?

Reply:
the syntax is the standard TOML configuration, and it is not relevant to this AutoEvent PR.  Please open another issue and we could enhance the configuration example to create multiple devices.

for the multiple encoding, since we have issue #233, why don't we resolve it in another PR?

Reply:
I just wanted you to confirm whether not the syntax supported multiple devices.

Re: multiple encoding let's keep that conversation in the relevant thread, this was about configuration only.